### DISCUSSION TITLE
Bugfixes or changes before proceeding with final merges? (development --> release --> master)

### DISCUSSION THREAD

**Discussion by SamuelTAFE** [D3nan/Tafe24S1T102_Ext#9] [General]
Hey all,

Raised a couple of topics regarding some small bugfixes we should be able to implement easily?

Is there anything else significant we should fix (either in the development or release branches) before merging a final copy back to master?

**Comment by D3nan** [comment]
I think all the assignment questions have been addressed and anything missing we can discuss in our Teams meeting. 
Once all the Teamwork sections have been done - I'll send out a copy of the report before submitting to make sure nothing is missing and the right screenshots are being captured.
